src/Main.java

```diff
@@ -1,37 +1,69 @@
 import java.util.Scanner;
+import java.util.Random;
 
 public class Main {
-
     public static void main(String[] args) {
-
-        // НИЖЧЕ ПРИКЛАД КОДУ, ВИКОРИСТОВУЙТЕ ЙОГО ЯК ЗРАЗОК,
-        // АЛЕ САМ ПРИКЛАД МОЖНА ВИДАЛИТИ ПОТІМ
-        // Приклад сканнера для перехоплення вводу користувача з консолі
-        System.out.println("Привіт, відважний герою!");
-        System.out.println("Тебе чекає цікава подорож");
-        System.out.println("Ти йшов чарівною стежкою, і натрапив на роздоріжжя: 1 - піти наліво, 2 - піти направо");
+        System.out.println("Ласквао просимо до казкового лісу!");
+        System.out.println("Ваше завдання - пройти ліс, ухвалюючи правильні рішення");
+        System.out.println("На старті Здоров'я=100, Удача=50 ");
         Scanner sc = new Scanner(System.in);
-        // Таким чином можна отримати цифру яку ввів користувач
-        // і зберегти її в змінну типу int
+        Random random = new Random();
+        int health = 100;
+        int luck = 50;
+// Action 1
+        System.out.println("Ви йшли чарівною стежкою, і натрапили на роздоріжжя: 1 - піти старою стежкою, 2 - обрати скарб серед кущів ");
+
         int choice1 = sc.nextInt();
         if (choice1 == 1) {
-            System.out.println("Ти обрав піти наліво");
-            // відняти очки здоров*я
+            System.out.println("Ви обрали піти старою стежкою");
+            health -= 20;
+            System.out.println("Ви пішли старою стежкою і втратили 20 очок здоров’я.");
         } else if (choice1 == 2) {
-            System.out.println("Ти обрав піти направо");
-            // додати очки удачі
-        } else {
-            System.out.println("Ти заплутався");
-            // відняти очки здоров*я
+            System.out.println("Ви обрали скарб серед кущів та отримали 30 очок удачі");
+            luck += 30;
+            System.out.println("Ви отримали скарб та 30 очок удачі! ");
+        } else if (choice1 > 2) {
```

Comment:
What if a user puts a negative number like `-10`?

Hint: there's a way to handle all ranges including the negative integers